packages/nimble-components/src/anchor-button/tests/anchor-button.mdx

```diff
@@ -0,0 +1,26 @@
+import {
+    Controls,
+    DocsStory,
+    Meta,
+    Stories,
+    Title,
+    Description
+} from '@storybook/blocks';
+import * as anchorButtonStories from './anchor-button.stories';
+
+<Meta of={anchorButtonStories} />
+
+<Title of={anchorButtonStories} />
+<Description of={anchorButtonStories} />
+
+<DocsStory of={anchorButtonStories.outlineAnchorButton} />
```

Comment:
I noticed that for the components we've converted to mdx docs instead of autodocs:
- their main story background is white instead of light gray; and
- the theme switcher at the top of the page doesn't update their main story background, so the dark/color themes don't render well:

![ComponentMdxThemeColorIssues](https://github.com/ni/nimble/assets/20709258/584741d0-3975-4baa-81b8-1a41cd2f3805)

After going through some Storybook code and experimenting, I think the easiest way to fix that issue is to use `DocsStory` instead of `Canvas` or `Story` for the main story in mdx docs.

Background info:
- The Storybook withBackground decorator, for docs view rendering, [creates a selector](https://github.com/storybookjs/storybook/blob/0792ef2db7e81a2719e24c9d587963eb8b4e8e47/code/addons/backgrounds/src/decorators/withBackground.ts#L38) that assumes that there will be an `Anchor` as the parent of the main story's `Canvas`. Our mdx docs aren't set up that way (but the autodocs are).
- However, we get that for free if we use [DocsStory](https://github.com/storybookjs/storybook/blob/265f21491c7c01cffaa5db569bd967185ff8d3c3/code/ui/blocks/src/blocks/DocsStory.tsx#L23), which has the `Anchor` and a child `Canvas` for the main story.